src/views/pages/index.ejs

```diff
@@ -17,7 +17,10 @@
             <li class="advantages-list-item">Зберігай та вивчай нові слова</li>
           </ul>
         </div>
-        <img class="hero-image" src="/assets/images/hero-image.png" alt="">
+        <div class="image-container">
+          <img class="hero-image" src="/assets/images/hero-image.png" alt="">
+          <div class="overlay"></div>
+        </div>
```

Comment:
As we use it already on 2 pages and in the future can use on more pages, I think it makes sense to make it reusable
However, as it's not customizable and will not receive any props, let's make in an include instead of the component